### PR TITLE
#159989005 setup empty react project

### PR DESCRIPTION
**Pull request by ascii-dev**
#### What does this PR do?
Setup empty react project

#### Description of Task to be completed?
- install and configure webpack
- install and configure babel
- install and configure react

#### How should this be manually tested?
- `git pull origin ch_159989005_setup_react`
- `npm install`
- `npm run start`

#### What are the relevant pivotal tracker stories?
[#159989005](https://www.pivotaltracker.com/story/show/159989005)

### CODE REVIEW

**Review by veeqtor — CHANGES_REQUESTED** [review]
I observed that there is no Eslint configuration yet, can you include that as well?.

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["env", "react"]
+}
```
Comment (anneKay) [inline .babelrc]: It might also be nice to add the `transform-class-properties` plugin to the .babelrc file

Comment (ascii-dev) [reply .babelrc]: Done! @veeqtor @anneKay

**Review by veeqtor — APPROVED** [review]

**Review by anneKay — APPROVED** [review]
It might also be nice to add the `transform-class-properties` plugin to the .babelrc file

**Review by anuonifade — APPROVED** [review]
OK

**Pull request merged**